### ISSUE TITLE
Fatal error after deploy log refactoring

### ISSUE THREAD

**Issue by t4deu** [meumobi/sitebuilder#54]
This fatal error has returned by the api after the deploy

*Fatal error: Uncaught exception 'UnexpectedValueException' with message 'The stream or file "/home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/log/sitebuilder.log" could not be opened: failed to open stream: Permission denied' in /home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/sitebuilder/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:84 Stack trace: #0 /home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/sitebuilder/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\Handler\StreamHandler->write(Array) #1 /home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/sitebuilder/vendor/monolog/monolog/src/Monolog/Logger.php(265): Monolog\Handler\AbstractProcessingHandler->handle(Array) #2 /home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/sitebuilder/vendor/monolog/monolog/src/Monolog/Logger.php(445): Monolog\Logger->addRecord(100, 'exception 'Unex...', Array) #3 /home/meumobi/PROJECTS/int-meumo in /home/meumobi/PROJECTS/int-meumobilesite.com/releases/20150415113636/sitebuilder/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 84*

This happened because during the deploy the file "log/sitebuilder.log" was created without write permissions for apache user.
And the API add a log on every access.

**Comment by juliogreff** [comment]
I'll fix the capistrano task, but did you grant permission on that file?

**Comment by t4deu** [comment]
yes, I did

**Comment by t4deu** [comment]
This issue is fixed? so I can close it, I din't see the changes in the capistrano task.

**Comment by t4deu** [comment]
The issue is not fixed, and after close the #139 change the capistrano task will not fix this issue.

To fix this update the lib/meumobi/sitebuilder/Logger.php to set the correct file permission(0666):
```php
class Logger
{
	public function __construct($level = null)
	{
		...
		$handler = new \Monolog\Handler\StreamHandler(APP_ROOT . '/log/sitebuilder-' . date('Y-m-d') . '.log', $level, true, 0666);
		...
	}
}
```

**Comment by juliogreff** [comment]
This has been fixed manually and will not happen again, since the files are already there. So I'm closing this one.
